synapse/lib/modelrev.py

```diff
@@ -757,6 +758,11 @@ async def revModel_0_2_24(self, layers):
         for form, props in formprops.items():
             await self._normVelocityProps(layers, form, props)
 
+    async def revModel_0_2_25(self, layers):
```

Comment:
This migration probably belongs in in the `_bumpCellVers('cortex:storage'` migrations in the Cortex rather than model migrations

Reply:
I did not know about that. I'll update.